services/py-api/src/service/hackathon_service.py

```diff
@@ -69,6 +69,22 @@ async def create_random_participant(
 
         # As when first created, the random participant is not assigned to a team we return the team as None
         return Ok((result.ok_value, None))
+    
+    async def create_invite_link_participant(
+            self, input_data: ParticipantRequestBody
+    ) -> Result[Tuple[Participant, Team], DuplicateEmailError | Exception]:
+        
+        participant_result = await self._participant_repo.create(input_data)
+        if is_err(participant_result):
+            return participant_result
```

Comment:
Make sure that this newly created participant has the field is_verified=True (participants with link are automatically verified and in the team_id field that have the id of the team with the same name as provided in the jwt token). In other words make sure they are present in the input data.

Reply:
Fixed in commit d8f0832f77647d818b784c5ad0942048bd62a834